strings/src/main/res/values/strings.xml

```diff
@@ -1065,4 +1065,7 @@
     <!-- Preceded by "Don't have a project yet?" question -->
     <string name="try_demo">Try a demo</string>
     <string name="scan_configuration_qr_code">Scan configuration QR code</string>
+
+    <!-- Error shown when a user tries to open a form that's in a different project -->
```

Comment:
Expressing this is tricky! An alternative — “To open this shortcut, you must first open ODK Collect and switch projects to the project that contains this form.”

Reply:
This message could be shown from an external app, not just a shortcut. I'm struggling to think of better text if I'm being honest!

Reply:
“To open this form, you must first open ODK Collect and switch to the project that contains it.”